### PR TITLE
Update /cube contact forms

### LINKED ISSUE #10481: [Update cube forms] add new fields to /cube/contact-us

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update /cube/contact-us and /cube#get-in-touch forms with new fields per the [brief](https://docs.google.com/document/d/14p8YZpe9z7H65pN_C5ebuVEajlLSgLU59zfIgepCwH8/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/contact-us and http://0.0.0.0:8001/cube#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the fields match the brief and that the form works

## Issue / Card

Fixes #10481 and #10482

## Screenshots

image

### CODE REVIEW

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Code looks good other than the small styling comment inline. 

QA failed as the form is failing to submit to Marketo. Sentry error: https://sentry.is.canonical.com/canonical/ubuntu-com/issues/21239/

**Comment by pmahnke** [comment]
@anthonydillon thanks, they are coming thru now

image

File: templates/shared/_cube-contact-us-form.html [hunk 1]
```diff
@@ -11,25 +11,105 @@ <h1>{{h1}}</h1>
...
+                <option value="Bash shell scripting">Bash shell scripting</option>
+                <option value="Performing administrative tasks">Performing administrative tasks</option>
+                <option value="Managing System Services">Managing System Services</option>
+                <option value="Networking on Ubuntu">Networking on Ubuntu</option>
+                <option value="Securing Ubuntu">Securing Ubuntu</option>
+                <option value="Ubuntu Kernels">Ubuntu Kernels</option>
+                <option value="Configuring enterprise storage">Configuring enterprise storage</option>
+                <option value="Virtualisation">Virtualisation </option>
+                <option value="MicroK8s">MicroK8s</option>
+                <option value="MaaS">MaaS</option>
```
Comment (anthonydillon) [inline templates/shared/_cube-contact-us-form.html:97]: Should be MAAS

File: templates/shared/_cube-contact-us-form.html [hunk 2]
```diff
@@ -11,25 +11,105 @@ <h1>{{h1}}</h1>
...
+              <label for="FirstName" id="LblFirstName"  >First name:</label>
+              <input id="FirstName" name="firstName" maxlength="255" aria-labelledby="LblFirstName" type="text" class="is-required" required="" aria-required="true" >
             </li>
-            <li class="mktFormReq mktField p-list__item">
-              <label for="lastName" class="mktoLabel">Last name:</label>
-              <input required id="lastName" name="lastName" maxlength="255" type="text" class="mktoField mktoRequired" />
+
+            <li  class="p-list__item  ">
+              <label for="LastName" id="LblLastName"  >Last name:</label>
+              <input id="LastName" name="lastName" maxlength="255" aria-labelledby="LblLastName InstructLastName" type="text" class="is-required" required="" aria-required="true" >
```
Comment (anthonydillon) [inline templates/shared/_cube-contact-us-form.html:21]: I thought these ids needed to be camel case instead of sentence case?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few questions in the code

File: templates/shared/_cube-contact-us-form.html (hunk 2 above)
Comment (pmahnke) [reply templates/shared/_cube-contact-us-form.html:21]: I thought only the `name=` mattered?

**Review by anthonydillon — APPROVED** [review]
Your right, QA worked fine. Thanks :+1:

**Pull request merged**